hypervisor/init_comm.go

```diff
@@ -55,7 +49,7 @@ func waitConsoleOutput(ctx *VmContext) {
 	}
 }
 
-func NewVmMessage(m *DecodedMessage) []byte {
+func NewVmMessage(m *hyperstartapi.DecodedMessage) []byte {
```

Comment:
It seems better if NewVmMessage() is also moved with a proper name. any suggestion?